cachetable.go

```diff
@@ -48,9 +48,6 @@ func (table *CacheTable) Count() int {
 
 // Foreach all items
 func (table *CacheTable) Foreach(trans func(key interface{}, item *CacheItem)) {
-	table.RLock()
```

Comment:
removed as a call to `store.Delete` in the scope of `store.Foreach` waits for the release of ReadLock and so it causes a deadlock #43

Reply:
Removing this lock will prevent a dead-lock, but it will break concurrency safety.